shared/map/render/Map.js

```diff
@@ -1,5 +1,5 @@
 import { Selection, Parcel } from '.'
-import { COLORS, getColor, getMapAsset } from '../../asset'
+import { COLORS, getColor } from '../'
```

Comment:
_Question_
I'm under the impression that doing this:

```javascript
import { Parcel } from './Parcel'
import { Selection } from './Selection'
import { COLORS, getColor } from '../Tile'
```

instead of 

```javascript
import { Selection, Parcel } from '.'
import { COLORS, getColor } from '../'
```

it's better to improve readability. For example, to write this very comment I had to check which files contained what.
I know the rest of `shared/map` uses this form so I'm not asking so we can change it, but rather to see what you think for later.

Same for https://github.com/decentraland/marketplace/pull/662/files#diff-24a4224e80ed9fc569fb73f3ead88f6b